### PR TITLE
Added Supporters for CoccinelleForRust

### PR DESCRIPTION
**Pull request by TathagataRoy1278**
I have added that Collabora is supporting CoccinelleForRust in its development.

Thank you!

### CODE REVIEW

File: src/Coccinelle-for-Rust.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ flow is not yet supported.
 
 [A recent talk about Coccinelle for Rust](https://gitlab.inria.fr/coccinelle/coccinelleforrust/-/blob/main/talks/rfl.pdf)
 
+## Support
```
Comment (ojeda) [inline src/Coccinelle-for-Rust.md:46]: In the commit title you mentioned "Sponsors" -- not sure if that word should be used somewhere here if that is the case.

File: src/Coccinelle-for-Rust.md [hunk 2]
```diff
@@ -43,6 +43,11 @@ flow is not yet supported.
...
+
+We would like to thank [Collabora](https://www.collabora.com/) for their support for
+the development of Coccinelle for Rust.
```
Comment (ojeda) [inline src/Coccinelle-for-Rust.md:49]:
```suggestion
We would like to thank [Collabora](https://www.collabora.com) for supporting
the development of Coccinelle for Rust.
```

**Review by ojeda — COMMENTED** [review]
I am merging this as-is and publishing it in a few minutes, but a couple suggestions for your future consideration, in case it helps.

File: src/Coccinelle-for-Rust.md (hunk 1 above)
Comment (TathagataRoy1278) [reply src/Coccinelle-for-Rust.md:46]: I am sorry, I should not have used that word. Thank you!

File: src/Coccinelle-for-Rust.md (hunk 2 above)
Comment (TathagataRoy1278) [reply src/Coccinelle-for-Rust.md:49]: Thank you!

Comment (ojeda) [reply src/Coccinelle-for-Rust.md:49]: Since you were here, if you want this suggestion to be applied, I can do it before publishing.

Comment (ojeda) [reply src/Coccinelle-for-Rust.md:49]: (And you are welcome! :)

Comment (TathagataRoy1278) [reply src/Coccinelle-for-Rust.md:49]: Yes, please. That would be great!
Thank you!

Comment (ojeda) [reply src/Coccinelle-for-Rust.md:49]: It is now live -- please take a look. Thanks!

Comment (TathagataRoy1278) [reply src/Coccinelle-for-Rust.md:49]: Its is perfect. Thank you!

**Pull request merged**